torba/torba/client/basenetwork.py

```diff
@@ -58,6 +59,7 @@ def connection_lost(self, exc):
 class BaseNetwork:
 
     def __init__(self, ledger):
+        asyncio.get_event_loop().set_exception_handler(protocol_exception_handler)
```

Comment:
Setting a global state like this in a random place is really bad. Also, there are other places where this is changed to another exception handler :man_facepalming: (although in the other case it's the wallet server, but still). This should be set in lbry `cli.py` or maybe `Daemon.py` or another place where it can be configurable and highly visible.